src/components/application_manager/src/application_data_impl.cc

```diff
@@ -316,18 +308,55 @@ const smart_objects::SmartObject* DynamicApplicationDataImpl::menu_icon()
   return menu_icon_;
 }
 
-const smart_objects::SmartObject* DynamicApplicationDataImpl::day_color_scheme()
+smart_objects::SmartObject DynamicApplicationDataImpl::day_color_scheme()
```

Comment:
Isnt the color scheme addressable to different window ids? Should this method take the window id as a parameter to return the proper color scheme?

Reply:
Never mind, I see the overloaded day_color_scheme method below